smoke-tests/designer/features/steps/steps.js

```diff
@@ -115,55 +117,53 @@ Then(
 
 When("I enter the details for my page", () => {
   this.newPageName = "Personal Details";
-  EditPageSection.pageTitle.setValue(this.newPageName);
-  EditPageSection.saveBtn.click();
+  editPage.pageTitle.setValue(this.newPageName);
+  editPage.saveBtn.click();
 });
 
 When("I link the {string} to the {string}", (fromPage, toPage) => {
   console.log(fromPage, toPage);
```

Comment:
do you need this anymore? can it be removed?

Reply:
@jenbutongit good spot removed